src/Components/Playlist modal/playlist-modal.css

```diff
@@ -0,0 +1,120 @@
+.playlist--section {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
```

Comment:
I know this works. But you could've even used `inset: 0` shorthand here.

Reply:
Thanks for the suggestion!